packages/documentation/src/content/docs/telemetry/overview.md

```diff
@@ -60,18 +62,31 @@ If an ASE does not provide the necessary exchange rate for a transaction, the te
 
 ## Instrumentation
 
-Rafiki currently has three counter metrics. All data points (counter increases and histogram records) are exported to collection endpoints at a configurable interval (default recommended to 15s).
+Rafiki currently has seven counter metrics. All data points (counter increases and histogram records) are exported to collection endpoints at a configurable interval (default recommended to 15s).
```

Comment:
This is kinda prone to drift from the truth (count, type of metrics). Feel like we could make our lives easier and avoid potentially misleading people by removing some of the incidental details and rephrasing like:

```suggestion
Data points for all metrics (e.g. counter increases) are exported to collection endpoints at a configurable interval (default recommended to 15s).
```

Currently there are more than 7 metrics (my bad - forgot to include the latest one I added 😬 ) and they arent all counters (`ilp_pay_time_ms` is a histogram for example).